packages/mui-base/src/AutocompleteUnstyled/useAutocomplete.js

```diff
@@ -1110,11 +1111,37 @@ export default function useAutocomplete(props) {
       tabIndex: -1,
       ...(!readOnly && { onDelete: handleTagDelete(index) }),
     }),
-    getListboxProps: () => ({
+    getListboxProps: ({ onScroll } = {}) => ({
       role: 'listbox',
       id: `${id}-listbox`,
       'aria-labelledby': `${id}-label`,
       ref: handleListboxRef,
+      onScroll: (event) => {
+        if (onScrollToBottom) {
+          const listboxNode = listboxRef.current;
+          const listboxHeight = listboxNode.offsetHeight;
+          const contentHeight = listboxNode.scrollHeight;
+          const scrollPosition = listboxNode.scrollTop;
+
+          const listboxPaddingBottom = Number(
```

Comment:
This looks suspicious, I suspect that we can simplify the logic by removing it.

Reply:
You mean calculating padding-bottom to be removed?

Reply:
i've explained my reasoning for calculating paddingBottom here (https://github.com/mui/material-ui/pull/35653#discussion_r1099033196)